lib/commands/certificate.js

```diff
@@ -80,8 +80,8 @@ async function clickElement (driver, locator, options = {}) {
   } = options;
   const lookupDelay = 500;
   try {
-    element = await retryInterval(timeout / lookupDelay, lookupDelay,
-       async () => await driver.findNativeElementOrElements(locator.type, locator.value, false)
+    element = await retryInterval(timeout < lookupDelay ? 1 : timeout / lookupDelay, lookupDelay,
+      async () => await driver.findNativeElementOrElements(locator.type, locator.value, false)
```

Comment:
This can be reduced to

```js
element = await retryInterval(timeout < lookupDelay ? 1 : timeout / lookupDelay, lookupDelay,
  driver.findNativeElementOrElements.bind(driver), locator.type, locator.value, false);
``